README.md

```diff
@@ -1,134 +1,124 @@
-  - [`useLockBodyScroll`](./docs/useLockBodyScroll.md) &mdash; lock scrolling of the body element.
-  - [`useSessionStorage`](./docs/useSessionStorage.md) &mdash; manages a value in `sessionStorage`.
-  - [`useThrottle` and `useThrottleFn`](./docs/useThrottle.md) &mdash; throttles a function. [![][img-demo]](https://streamich.github.io/react-use/?path=/story/side-effects-usethrottle--demo)
-  - [`useTitle`](./docs/useTitle.md) &mdash; sets title of the page.
-    <br/>
-    <br/>
-- [**Lifecycles**](./docs/Lifecycles.md)
-  - [`useEffectOnce`](./docs/useEffectOnce.md) &mdash; a modified [`useEffect`](https://reactjs.org/docs/hooks-reference.html#useeffect) hook that only runs once.
-  - [`useEvent`](./docs/useEvent.md) &mdash; subscribe to events.
-  - [`useLifecycles`](./docs/useLifecycles.md) &mdash; calls `mount` and `unmount` callbacks.
-  - [`useRefMounted`](./docs/useRefMounted.md) &mdash; tracks if component is mounted.
-  - [`usePromise`](./docs/usePromise.md) &mdash; resolves promise only while component is mounted.
-  - [`useLogger`](./docs/useLogger.md) &mdash; logs in console as component goes through life-cycles.
-  - [`useMount`](./docs/useMount.md) &mdash; calls `mount` callbacks.
-  - [`useUnmount`](./docs/useUnmount.md) &mdash; calls `unmount` callbacks.
-  - [`useUpdateEffect`](./docs/useUpdateEffect.md) &mdash; run an `effect` only on updates.
-    <br/>
-    <br/>
-- [**State**](./docs/State.md)
-  - [`createMemo`](./docs/createMemo.md) &mdash; factory of memoized hooks.
-  - [`useGetSet`](./docs/useGetSet.md) &mdash; returns state getter `get()` instead of raw state.
-  - [`useGetSetState`](./docs/useGetSetState.md) &mdash; as if [`useGetSet`](./docs/useGetSet.md) and [`useSetState`](./docs/useSetState.md) had a baby.
-  - [`useObservable`](./docs/useObservable.md) &mdash; tracks latest value of an `Observable`.
-  - [`useSetState`](./docs/useSetState.md) &mdash; creates `setState` method which works like `this.setState`. [![][img-demo]](https://codesandbox.io/s/n75zqn1xp0)
-  - [`useToggle` and `useBoolean`](./docs/useToggle.md) &mdash; tracks state of a boolean.
-  - [`useCounter` and `useNumber`](./docs/useCounter.md) &mdash; tracks state of a number.
-  - [`useList`](./docs/useList.md) &mdash; tracks state of an array.
-  - [`useMap`](./docs/useMap.md) &mdash; tracks state of an object.
-    <br/>
-    <br/>
-
-<br />
-<br />
-<br />
-
-<h2 align="center"><sub>Usage</sub></h2>
-
-<br/>
-
-<p align="center">
-You need to have React <code>16.8.1</code> or later installed to use Hooks API.
-</p>
-
-<p align="center">
-You can import each hook individually <code>import useToggle from 'react-use/lib/useToggle'</code>.
-</p>
-
-<br/>
-
-<h2 align="center"><sub>License</sub></h2>
-
-<p align="center">
-  <a href="./LICENSE">Unlicense</a> &mdash; public domain.
-</p>
-
-[img-demo]: https://img.shields.io/badge/demo-%20%20%20%F0%9F%9A%80-green.svg
+# 👍 react-use
```

Comment:
I'm not sure I would want to do layout changes in README (though, badges can be added and `babel-plugin-import` possibly, too). The README does look different in NPM, because NPM does not center the text, but the spacing and centering was designed specifically for GitHub, I'm not sure I would want to change it now.